### ISSUE TITLE
Please show error messages

### ISSUE THREAD

**Issue by antm** [woollyapp/woolly#8]
If there's an error in the query the only indication is the failure message at the top - displaying the message in the result pane would be really helpful for debugging and editing.

**Comment by peter-assi** [comment]
:heavy_plus_sign:  :100:

**Comment by stevenrombauts** [comment]
Agreed as well. This little feature would make all the difference and save us loads of time. + ∞
